modulemd/v2/tests/test-modulemd-merger.c

```diff
@@ -58,6 +58,8 @@ merger_test_deduplicate (CommonMmdTestFixture *fixture,
   index = modulemd_module_index_new ();
 
   g_assert_nonnull (yaml_path);
+  g_assert_nonnull (index);
+  g_assert_null (error);
```

Comment:
Checking the `error` here is meaningless, as the `error` was initialized to NULL and no function has been called that might modify it.

Reply:
Okay. I shall contribute accordingly. Thank you for the guidance.